cmd/cassandra-writer/main.go

```
@@ -108,43 +108,18 @@ func loadConfig() config {
 		Port:     dbPort,
 	}
 
-	chanCfgPath := mainflux.Env(envChanCfgPath, defChanCfgPath)
+	channelsCfgPath := mainflux.Env(envChannelsCfgPath, defChannelsCfgPath)
+	subtopicsCfgPath := mainflux.Env(envSubtopicsCfgPath, defSubtopicsCfgPath)
+	filters, _ := writers.LoadFiltersConfig(channelsCfgPath, subtopicsCfgPath)
```

Don't ignore errors please